scripts/azure-pipelines/linux-build_and_test.sh

```diff
@@ -47,8 +47,8 @@ if [[ "$DO_BUILD" == "yes" ]] ; then
     fi
 fi
 
-find ./axom/sidre -type d -exec chmod 755 {} \;
-find ./axom/sidre -type f -exec chmod 644 {} \;
+find ./axom -type d -exec chmod 755 {} \;
```

Comment:
Thanks for fixing this!

Should we add a comment about what these lines are doing?

Reply:
Maybe something like this...
```suggestion
# Note: Azure pipelines requires read/write access for everyone between steps
find ./axom -type d -exec chmod 755 {} \;
```